r/tests/testthat/test-parquet.R

```diff
@@ -25,10 +25,3 @@ test_that("reading a known Parquet file to tibble", {
   expect_identical(dim(df), c(10L, 11L))
   # TODO: assert more about the contents
 })
-
-test_that("as_tibble with and without threads", {
```

Comment:
Let's keep this test so that we're exercising the single-threaded code and ensuring equality. Just have to change the `use_threads=FALSE` to flip the option instead.